Tools/MU-CSeq.md

```diff
@@ -31,6 +31,10 @@ MU-CSeq 0.4/Artifact of TACAS '16 paper: http://users.ecs.soton.ac.uk/gp4/cseq/m
 Project page: https://github.com/CSeq/Overview#mu-cseq
 
 #### Last commit date:
+29 May 2021 (last activity)
+29 May 2021 (last activity)
+29 May 2021 (last activity)
+29 May 2021 (last activity)
```

Comment:
Any idea why this is duplicated so often?

Reply:
Yes, this is a bug in the aggregator due to a missing cleanup. Affect 4 tools and all solved